svc-bie-kafka/src/main/java/gov/va/vro/services/bie/service/kafka/MessageHelper.java

```diff
@@ -0,0 +1,22 @@
+package gov.va.vro.services.bie.service.kafka;
+
+import gov.va.vro.model.biekafka.ContentionEvent;
+
+import java.util.Arrays;
+
+public class MessageHelper {
+  public static ContentionEvent mapTopicToEvent(String topic) {
+    // remove first word prefix from topic seperated by _
+    String noPrefixTopic = topic.substring(topic.indexOf("_") + 1);
```

Comment:
True, now that this is moved into the `ContentionEvent` which is in a shared library, it doesn't have access to the bie properties. Passing in as a variable seems a bit ugly overall, since java doesn't support default params. Let me create another issue to get this resolved.